src/common/PropertyExt.js

```diff
@@ -266,6 +266,12 @@
         for (var i in clsArr) {
             if (theme[clsArr[i]]) {
                 for (var paramName in theme[clsArr[i]]) {
+                    if (paramName === "overrideTags" && theme[clsArr[i]][paramName] instanceof Object) {
+                        for (var param in theme[clsArr[i]][paramName]) {
+                            this.tags(param,theme[clsArr[i]][paramName][param]);
```

Comment:
*this.tags* method doesn't need to exist (its only used here). It could be replaced with this:
```javascript
if(this.publishedProperty(param).ext){
    this.publishedProperty(id).ext.tags = theme[clsArr[i]][paramName][param];
}
```